# GIT_Atividade_William_01
Atividade de Versionamento do William Brito

Passo 01
---------
Criação dos arquivos
* Codigo_01_input.txt  
* Codigo_02_Banco_de_Dados.txt


Passo 02
--------
Foi feito a sincronização com GitHUB


Passo 03
--------
Crição do arquivo
* Codigo_03_Processamento


Passo 04
--------
Foi feito a sincronização com GitHUB


Passo 05
--------
Criação do Readme.md dentro do Github
 

Passo 06
--------
Alteração no Codigo_03_Processamento para inclusão de processamento de fornecedores

Passo 07
-------
Foi feito a sincronização com GitHUB

Mas ocorreu erro:
$ git push
To https://github.com/WilliamBritoFreitas/GIT_Atividade_William_01.git
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'https://github.com/WilliamBritoFreitas/GIT_Atividade_William_01.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Contornando o problema fazendo um pull do readme.md para máquina local.

Passo 08
--------

Foi feito commit da Adição de processamento de Fornecedores

E feito push

